docs/OV_Converter_UG/prepare_model/convert_model/MO_OVC_transition.md

```diff
@@ -0,0 +1,634 @@
+   * - share_weights
+     - share_weights
+     - No differences.
+   * - framework
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - help / -h
+     - help / -h
+     - OVC provides help parameter only in command line tool.
+   * - example_output
+     - output
+     - OVC / ov.convert_model 'output' parameter includes capabilities of MO 'example_output' parameter.
+   * - input_model_is_text
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - input_checkpoint
+     - input_model
+     - All supported model formats can be passed to 'input_model'.
+   * - input_meta_graph
+     - input_model
+     - All supported model formats can be passed to 'input_model'.
+   * - saved_model_dir
+     - input_model
+     - All supported model formats can be passed to 'input_model'.
+   * - saved_model_tags
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - tensorflow_custom_operations_config_update
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - tensorflow_object_detection_api_pipeline_config
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - tensorboard_logdir
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - tensorflow_custom_layer_libraries
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - input_symbol
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - nd_prefix_name
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - pretrained_model_name
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - save_params_from_nd
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - legacy_mxnet_model
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - enable_ssd_gluoncv
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - input_proto
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - caffe_parser_path
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - k
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - disable_omitting_optional
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - enable_flattening_nested_params
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - counts
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - remove_output_softmax
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+   * - remove_memory
+     - N/A
+     - Not available in ov.convert_model() / OVC.
+
+Transition from legacy API to new API
+############################################################################
+
+mo.convert_model() provides a wide range of preprocessing parameters. Most of these parameters have analogs in OVC or can be replaced with functionality from ``ov.PrePostProcessor`` class.
+Here is the guide to transition from legacy model preprocessing to new API preprocessing.
+
+
+``input_shape``:
```

Comment:
this might seem unnecessary but is done for the sake of the right hand menu http://openvino-doc.iotg.sclab.intel.com/mo_docs/openvino_docs_OV_Converter_UG_prepare_model_convert_model_MO_OVC_transition.html#input-shape